src/Sarif.Viewer.VisualStudio.Core/CodeAnalysisResultManager.cs

```diff
@@ -683,6 +685,72 @@ internal SarifLog GetPartitionedLog(IEnumerable<SarifErrorListItem> listItems)
             return partitions[guid];
         }
 
+        internal void AddSuppressionToSarifLog(SuppressionModel suppressionModel)
+        {
+            if (suppressionModel?.SelectedErrorListItems?.Any() != true)
```

Comment:
>.Any() != true [](http://example.com/codeflow?start=57&length=14)

question, do VS recommend !Any() or  Any() != true

Reply:
this is because the '?' operator, the left side of the equation can be null or a boolean. cannot use '!' operator since it cannot be used for null value. but both null and boolean can be compared with another boolean.